subsys/mgmt/mcumgr/transport/src/smp_udp.c

```diff
@@ -398,6 +388,9 @@ static void smp_udp_start(void)
 #endif
 
 #ifdef CONFIG_MCUMGR_TRANSPORT_UDP_IPV6
+	configs.ipv4.proto = PROTOCOL_IPV6;
+	configs.ipv4.sock = -1;
+
```

Comment:
This is possibly falsely reducing usage because there are not the right fields

Reply:
@nordicjm Sorry, I'm not sure what you mean?

The ROM usage is reduced because by initialising the fields here rather than statically, the `configs` struct can be statically zero-initialised and doesn't need to go in ROM at all.

Reply:
These are `.ipv4` but this is in the IPv6 section